src/main.js

```diff
@@ -112,3 +113,9 @@ function displaySavedCovers() {
       </section> `)
   }
 }
+
+function deleteSavedCover(event) {
+  var closestCover = event.target.closest('.mini-cover');
```

Comment:
I'm not sure of a better way to click on the mini-cover . The event.target.closest is grabbing that dbl click event and when this happens, the closest mini cover class will be deleted.